www/src/pages/index.js

```diff
@@ -19,10 +19,79 @@ import FuturaParagraph from "../components/futura-paragraph"
 import Button from "../components/button"
 import TechWithIcon from "../components/tech-with-icon"
 import EmailCaptureForm from "../components/email-capture-form"
+import HomepageEcosystem from "../components/homepage/homepage-ecosystem"
+import {
+  setupScrollersObserver,
+  unobserveScrollers,
+} from "../utils/scrollers-observer"
 
 class IndexRoute extends React.Component {
+  componentDidMount() {
+    setupScrollersObserver()
+  }
+
+  componentWillUnmount() {
+    unobserveScrollers()
+  }
+
+  combineEcosystemFeaturedItems = ({ starters, plugins }) => {
+    const combinedItems = []
```

Comment:
Just a suggestion:

```js
return new Array(3).fill(undefined).reduce((merged, _, index) => {
  return merged.concat([
    starters[index],
    plugins[index]
  ])
}, [])
```

Reply:
Oh and maybe do something like this, too?

```jsx
combineEcosystemFeaturedItems = ({ starters, plugins }, numFeatured = 3) => {
 // use numFeatured instead of "3"
```

Reply:
:+1: